### ISSUE TITLE
Raspberry pi External Trigger

### ISSUE THREAD

**Issue by habib-Boloorchi** [ArduCAM/ArduCAM_USB_Camera_Shield#66]
I want to manipulate a camera to give me image 20 fps by external trigger but it made for 2 cameras and I just want to use one camera and an Imu with different timestamps.  would you tell me which parameters have to be changed to do this job?
Would you tell me how to change it to make a Rosbag from it?

**Comment by Razbotics** [comment]
Exactly same thing i want to do. Did you achieve it? I will buy the camera if external triggering works.

**Comment by habib-Boloorchi** [comment]
The trigger works but the data wont come to raspberry consistantly. It has too much delay and delays are change. It is not stable. I am keep searching to find a better choice.

**Comment by Razbotics** [comment]
Did u tried with your laptop or pc?

**Comment by habib-Boloorchi** [comment]
I will try it next week, and call u back.

**Comment by Razbotics** [comment]
Yes please do it and let me know. So that i can buy the camera module
